scripts/localize.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Check if environment variables have been defined for our team ID, AAD App ID, and AAD App Secret for automated workflows before requesting them interactively
+if [ -z $TDBUILD_TEAM_ID ]; then
+    printf "Team ID: "
+    read TDBUILD_TEAM_ID
+fi
+
+if [ -z $TDBUILD_AAD_APPLICATION_CLIENT_ID ]; then
+    printf "Alias: "
+    read TDBUILD_AAD_APPLICATION_CLIENT_ID
+fi
+
+if [ -z $TDBUILD_AAD_APPLICATION_CLIENT_SECRET ]; then
+    stty -echo
+    printf "Password: "
+    read TDBUILD_AAD_APPLICATION_CLIENT_SECRET
+    stty echo
+    printf "\n"
+fi
+
+# Localize iOS resources
+scripts/GetLocalizedFiles.sh -t $TDBUILD_TEAM_ID -u -a $TDBUILD_AAD_APPLICATION_CLIENT_ID -p $TDBUILD_AAD_APPLICATION_CLIENT_SECRET -f ios/FluentUI/Resources/Localization/en.lproj -r ios/Localization/en.lproj -o ios/FluentUI/Resources/Localization
+
+# Localize macOS resources
+scripts/GetLocalizedFiles.sh -t $TDBUILD_TEAM_ID -u -a $TDBUILD_AAD_APPLICATION_CLIENT_ID -p $TDBUILD_AAD_APPLICATION_CLIENT_SECRET -f macos/FluentUIResources-macos/Strings/en.lproj -r macos/Localization/en.lproj -o macos/FluentUIResources-macos/Strings
```

Comment:
@anandrajeswaran @harrieshin, the `-r macos/Localization/en.lproj` is the relative path that these strings should be stored under on the tdbuild localization server. It doesn't actually need to correspond to any paths in our repo. This allows us to be referencing the same localization files on the service but still be able to move/refactor our code in the repo so a moved file doesn't trigger relocalization of all the strings. 

Do you all like the <platform>/Localization/en.lproj pattern?

Reply:
I've looked into it a bit more, and based on other examples, it probably makes the most sense to get rid of everything other than just `platform_name`. Other examples don't provide the en.lproj, and the Localization relative directory just kind of feels implied.